GiEnJul/ClientApp/src/components/institution/InstitutionPersonInterface.tsx

```diff
@@ -0,0 +1,12 @@
+import Gender from "../../common/enums/Gender";
+
+interface IInstitutionPerson {
```

Comment:
The name IInstitutionPerson seems confusing, as the properties are connected to a recipient and not a submitter?